arduino/WandCore/TouchSurfaceClient.cpp

```diff
@@ -0,0 +1,115 @@
+/*
+  Capacitive Touch Surface Client
+
+  Manages a Touch Surface Input
+*/
+
+#include "TouchSurfaceClient.h"
+
+// Pin the touch surface is attached to
+int touchPin;
+
+// Touch surface constants
+const int touchThold = 50;                  // readings above mean the sensor is not being touched

```

Comment:
I read this as `touch` `T` `hold` when I first saw it, I think I would name it `touchThresh` or `touchThreshold`.